devtools/common/src/main/java/io/quarkus/generators/SourceType.java

```diff
@@ -17,6 +17,7 @@
 
     private static final String BUILD_FILE_RESOURCE_TEMPLATE = "templates/%s/%s/%s-template.ftl";
     private static final String RESOURCE_TEMPLATE = "templates/%s/%s/resource-template.ftl";
+    private static final String SPRING_CONTROLLER_TEMPLATE = "templates/%s/%s/spring-resource-template.ftl";
```

Comment:
I think the template file should be called `spring-controller-template`, don't you?